src/sensordatainterface/views/api_views.py

```diff
@@ -26,10 +26,23 @@ def get_equipment_by_site(request):
     if request.method == 'POST':
         site_selected = request.POST.get('site_selected')
 
-        equipment_deployed = Equipment.objects.filter(
-            equipmentused__actionid__featureaction__samplingfeatureid=site_selected
-        ).distinct()
+        action_type = request.POST.get('formType')
 
+        if action_type.find("Instrument") > -1:

```

Comment:
you can do 
`is_instrument = action_type.find("Instrument") > -1
equipment_deployed = Equipment.objects\
    .filter(equipmentused__actionid__featureaction__samplingfeatureid=site_selected)\
    .distinct()\
    .filter(equipmentmodelid__isinstrument=is_instrument)
`